robottelo/cli/hostcollection.py

```diff
@@ -50,6 +50,11 @@ def transform_ids(cls, options=None):
             ]
             if host_ids:
                 options['host-ids'] = ','.join(host_ids)
+        if 'host-id' in options:
+            host_id = options.get('host-id')
+            if host_id and not host_id.isdigit():
+                host_name = ContentHost.info({'id': host_id})['name'].lower()
+                options['host-id'] = Host.info({'name': host_name})['id']
```

Comment:
I've updated `host_id` part that was added in this PR, however entire refactoring for this function is slightly out of scope.